CONTRIBUTING.md

```diff
@@ -26,6 +26,7 @@ atrophius:
     bio: Here's a bio for me.
     url: http://www.jeremyprivett.com
     twitter: Atrophius
+    github: Atrophius
```

Comment:
Im wondering if we can eliminate the need for this line since the "key" is technically the same username as github by convention.